README.md

```diff
@@ -20,8 +20,8 @@ Any option without a default is required.
 | `build-backend`       | The wrapped build backend (e.g. `setuptools.build_meta`)            | string         |                     | N                             |
 | `commit-file`         | The file in which to write the git commit hash                      | string         | "" (No file)        | N                             |
 | `dependencies-file`   | The path to the `dependencies.yaml` file to use                     | string         | "dependencies.yaml" | Y                             |
+| `disable-cuda`        | If true, CUDA version is not used in package name or dependencies   | bool           | false               | Y                             |
```

Comment:
```suggestion
| `disable-cuda`        | If true, CUDA version in build environment is ignored when setting package name or dependencies   | bool           | false               | Y                             |
```

Would you consider a re-write like this?

CUDA version is "not used" sounds to me like if I had a project name *that already had a CUDA suffix like `-cu12`*, setting `disable-cuda: true` might cause `rapids-build-backend` to strip that suffix off.

I think something like what I'm proposing here makes the exact effect a bit clearer.

If you disagree, feel free to disregard this comment. It's a minor personal preference.